code/processes/tickerlogreplay.q

```diff
@@ -332,12 +333,17 @@ merge:{[dir;pt;tablename;mergelimits;h]
  };
 
 
+
+
 \d .
 //load the sort csv
 .sort.getsortcsv[.replay.sortcsv]
 
-.replay.replaylog each .replay.logstoreplay;
-.lg.o[`replay;"replay complete"]
-if[.replay.exitwhencomplete; exit 0]
 
 
+if[.replay.autoreplay;.lg.o[`replay;"replay starting from script by default"];
```

Comment:
I have to say this is one of the few things that annoys me. The first of the if statement contents being on a new line means its not @simple@ to tell if its an endline or something else at a glance.

[ But I enter this because comment  I wanted to see how Git works as much as anything else. ]

Sam change to suit the Gitstapo

Reply:
For me having the logging on this line implies it is the only action happening as a result of this condition. 

When there are multiple actions I prefer to see them on newlines to highlight each statement.

Reply:
Surely the closing brace at the end of the line would tell you that (and that's what your eye would be looking for. Anyway, its not a point Id be prepared to argue over as its a waste of time. 

I would say though that code check ins at some of the public projects must be fun !!!

Reply:
logging moved to its ownline